MANIFEST.in

```diff
@@ -7,8 +7,10 @@ include CONTRIBUTING.md
 exclude .coveragerc
 exclude .pylintrc
 exclude tox.ini
+exclude noxfile.py
 exclude requirements.txt
 exclude requirements-dev.txt
+exclude pip/*
```

Comment:
This seems wrong; you're committing files in pip/ in this commit IIUC.

Reply:
right, but we don't need the requirements files in the dist do we? Same as requirement.txt and requirements-dev.txt in the preceeding lines